pkg/detectors/ayrshare/ayrshare.go

```diff
@@ -21,7 +22,7 @@ var (
 	client = common.SaneHttpClient()
 
 	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
-	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"ayrshare"}) + `\b([A-Z]{7}-[A-Z0-9]{7}-[A-Z0-9]{7}-[A-Z0-9]{7})\b`)
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"ayrshare"}) + `\b([A-Z0-9]{8}-[A-Z0-9]{8}-[A-Z0-9]{8}-[A-Z0-9]{8})\b`)
```

Comment:
Has the Ayrshare secret pattern been updated? This is a critical change and should only be made once we are certain that the secret pattern has been updated correctly.

Reply:
I generated three API keys on [AyrShare](https://app.ayrshare.com/api), and it seems they've updated the pattern. The old regex is no longer detecting the keys.

Reply:
Yes, I too tried with different keys, seems like they are using 8 characters now instead of 7,